cmd/podman/create.go

```diff
@@ -626,19 +632,32 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime, imageName string,
 		return nil, errors.Errorf("cannot pass additional search domains when also specifying '.'")
 	}
 
+	ImageVolumes := []string{}
+	for k := range data.ContainerConfig.Volumes {
+		ImageVolumes = append(ImageVolumes, k)
+	}
+
+	if c.IsSet("image-volumes") {
+		if c.String("image-volumes") != "bind" && c.String("image-volumes") != "tmpfs" && c.String("image-volumes") != "ignore" {
```

Comment:
Can you add a map, and do it that way.
if ! valid_imagevolumes[c.String("image-volumes") {
}

Reply:
fixed